### PR TITLE
fix: fixed new localization to properly load only default language

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
Seems like append did not properly work. Now when no language was found the default language is properly returned.

### CODE REVIEW

**Comment by Luphoxa** [comment]
Test Results:
- Shows the instructions as indicated in Creator_DE_desktop_mode 👍 
- Changes in Creator_DE_desktop_mode are applied the next project start 👍 
- I can't change languages somehow. My tts setting is on DE, but my project is in EN and I can't change it 👎

**Comment by tomwim** [comment]
List has both languages from the Localization folder and the one from the Config
image

**Comment by tomwim** [comment]
Having no Localiazation file and setting the language in the config works

**Comment by tomwim** [comment]
Both of my reported bugs were fixed. Thank you so much @SimonTheSourcerer

**Comment by Luphoxa** [comment]
Test Notes:
👍 Language put to EN or DE shows the corresponding hover text in EN or DE  
👍  Changing to a language that has no desktop mode txt file, the default desktop text is shown 
👎 Creating a language file for a new language (I copied and renamed the EN file and changed the text) and setting this language in the language settings does change it correctly for TTS, but the desktop mode still uses the default txt file, although a file for it exists. Maybe I forgot to add something? Need to have a look here, maybe together if you want  

Will add more results if I find something else :)

File: Runtime/Localization/LanguageSettings.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Innoactive.Creator.Core.Runtime.Utils;
+
+namespace Innoactive.Creator.Core.Internationalization
+{
+    public class LanguageSettings : SettingsObject<LanguageSettings>
```
Comment (Gusinuhe) [inline Runtime/Localization/LanguageSettings.cs:6]: Missing docs

**Review by Gusinuhe — DISMISSED** [review]

File: Editor/UI/ProjectSettings/LanguageSettingsProvider.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿using Innoactive.Creator.Core.Internationalization;
+using UnityEditor;
+
+namespace Innoactive.CreatorEditor.UI
+{
+    public class LanguageSettingsProvider : BaseSettingsProvider
```
[the remark this answers is not in the document]
Comment (SimonTheSourcerer) [reply Editor/UI/ProjectSettings/LanguageSettingsProvider.cs:6]: yes~

**Comment by tomwim** [comment]
Switching languages during runtime does not change the spoken TTS language. The text is changed

File: Runtime/Localization/LanguageSettings.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    /// </summary>
+    public class LanguageSettings : SettingsObject<LanguageSettings>
+    {
+        /// <summary>
+        /// Language which should be used.
+        /// </summary>
+        public string DefaultLanguage = "En";
+
+        /// <summary>
+        /// Returns the active, will be stored for one session.
```
Comment (tomwim) [inline Runtime/Localization/LanguageSettings.cs:16]:
```suggestion
        /// Returns the currently active language, will be stored for one session.
```

**Review by tomwim — APPROVED** [review]

**Review by Gusinuhe — APPROVED** [review]

**Pull request merged**